### ISSUE TITLE
Dashboard links

### ISSUE THREAD

**Issue by AntoineThebaud** [perses/perses#1642]
### Proposal

Today we are lacking a way to quickly jump from a dashboard to another with already-set variables. Thus it would be great to have links as part of the dashboard data model, ideally both at top level & at panels level.

**Comment by sjcobb** [comment]
Yeah this will be really useful. Do you see passing variable context as a checkbox when setting up the link, is it on by default? Do we allow passing selected time range too? 

Also curious how you see global and project level variables interacting with this, should those values stay selected when moving between dashboards?

**Comment by sjcobb** [comment]
Also the tags discussion is relevant, how often do you see that feature used? Where you setup a dashboard link, pick a tag, and have the links be dynamic

**Comment by AntoineThebaud** [comment]
Yes, checkboxes to (dis)able passing time range & variable parameters would be a must, and regarding variables I suppose it shouldn't differentiate between the different scopes.

And yes the day we have tags, we probably want to allow generating links for dashboards matching a given tags selection. We can also imagine the same kind of option but passing a (sub-)folder instead/at the same time?

On our side both the the top-level & panel-level links are used in important dashboards.

**Comment by prakhar29jain** [comment]
I will take this up!

**Comment by celian-garcia** [comment]
Hello, just adding for reference 
https://github.com/perses/shared/pull/67
https://github.com/perses/perses/pull/3905

@prakhar29jain please reference next time the issue in your PRs <3 don't hesitate to close if done
